package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "eslint-plugin-blackbird",
+  "version": "0.0.1",
+  "description": "Blackbird ESLint rules",
+  "keywords": [
+    "eslint",
+    "eslintplugin",
+    "eslint-plugin"
+  ],
+  "author": "Peter Stuart",
+  "main": "lib/index.js",
+  "scripts": {
+    "build": "babel src -d lib",
+    "test": "mocha lib/tests --recursive",
```

Comment:
Does this have to run against `lib/` (ie the compiled directory?) as opposed to `src/`? Seems inconvenient to have to run `yarn build` every time before `yarn test`

Reply:
Hmm, it does seem to need to run against `lib`, but I can just include `build` in the `test` script at least.